bin/start-vm

```diff
@@ -241,6 +241,7 @@ set_binaries () {
     bin_stat="stat"
     bin_sed="sed"
     bin_uuid="cat /proc/sys/kernel/random/uuid"
+    bin_realpath="grealpath"
```

Comment:
the `g` prefix is macos/homebrew specific, on linux the binary is called `realpath`

Reply:
good catch, yeah should be realpath for linux and grealpath on macos, having grealpath in both places was a typo